src/components/Login.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import axios from 'axios';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import { Container, Header, Button } from 'semantic-ui-react';
+
+const LoginSchema = Yup.object().shape({
+  // telling Formik what shape the input data is supposed to be
+  email: Yup.string()
+    .email('Email is not valid')
+    .required('Please enter your email address'),
+  password: Yup.string()
+    .required('Please enter a password')
+});
+
+function Login (props, { errors, touched, values, isSubmitting }) {
+  
+  return (
+    <Container className="container" >
+      <Header as="h1">Login</Header>
+        <Formik
+          initialValues={{ email: '', password: ''}}
+          validationSchema={LoginSchema}
+          onSubmit={( values, { resetForm, setErrors, setSubmitting, setStatus }) => {
```

Comment:
All you really need here is the `resetForm` and perhaps `setStatus` method for the catch statement and showing a login error.